service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseManagementOrderProgressionControllerTest.java

```diff
@@ -68,31 +96,88 @@ void aboutToSubmitShouldPopulateListServedCaseManagementOrdersWhenSendsToAllPart
+            .status(SEND_TO_JUDGE)
+            .id(uuid)
+            .action(OrderAction.builder()
+                .type(SEND_TO_ALL_PARTIES)
+                .build())
+            .build();
+
+        CaseDetails caseDetails = CaseDetails.builder()
+            .data(ImmutableMap.<String, Object>builder()
+                .putAll(caseDataMap(order))
+                .putAll(getNotificationData())
+                .build())
+            .id(CASE_ID)
+            .build();
+
+        postAboutToSubmitEvent(buildCallbackRequest(caseDetails, DRAFT_CASE_MANAGEMENT_ORDER));
+
+        verify(notificationClient, times(1)).sendEmail(
+            eq(CMO_READY_FOR_JUDGE_REVIEW_NOTIFICATION_TEMPLATE), eq("admin@family-court.com"),
+            eq(expectedTemplateParameters()), eq(CASE_REFERENCE));
+    }
+
+    private Map<String, Object> caseDataMap(CaseManagementOrder order) {
         return ImmutableMap.of(
             CASE_MANAGEMENT_ORDER_JUDICIARY.getKey(), order,
             HEARING_DETAILS_KEY, List.of(Element.<HearingBooking>builder()
                 .id(uuid)
                 .value(HearingBooking.builder()
-                    .startDate(localDateTime)
+                    .startDate(YESTERDAY)
                     .build())
                 .build()));
     }
 
-    private CallbackRequest buildCallbackRequest(CaseDetails caseDetails) {
+    private Map<String, Object> getNotificationData() {
+        return ImmutableMap.of(
+            "cmoEventId", "draftCMO",
+            "caseLocalAuthority", LOCAL_AUTHORITY_CODE,
+            "familyManCaseNumber", FAMILY_MAN_CASE_NUMBER,
+            "respondents1", ImmutableList.of(
+                    ImmutableMap.of(
+                    "value", Respondent.builder()
+                        .party(RespondentParty.builder()
+                            .lastName(RESPONDENT_SURNAME)
+                            .build())
+                        .build()
+                    ))
+        );
+    }
+
+    private CallbackRequest buildCallbackRequest(CaseDetails caseDetails, Event event) {
         return CallbackRequest.builder()
-            .eventId(ACTION_CASE_MANAGEMENT_ORDER.getId())
+            .eventId(event.getId())
             .caseDetails(caseDetails)
             .build();
     }
 
+    private Map<String, Object> expectedTemplateParameters() {
+        return ImmutableMap.<String, Object>builder()
+            .put("subjectLineWithHearingDate", buildSubjectLine())
+            .put("reference", CASE_REFERENCE)
+            .put("respondentLastName", RESPONDENT_SURNAME)
+            .put("caseUrl", "http://fake-url/case/" + JURISDICTION + "/" + CASE_TYPE + "/12345")
+            .build();
+    }
+
+    private String buildSubjectLine() {
+        String hearingDate = dateFormatterService
+            .formatLocalDateTimeBaseUsingFormat(YESTERDAY, "d MMM yyyy");
+
+        return String.format("%s, %s, hearing %s", RESPONDENT_SURNAME, FAMILY_MAN_CASE_NUMBER, hearingDate);
+    }
 }
```

Comment:
would we need a test scenario for when we do not have a value for `hearingDate`? Or would we always have this value?

Reply:
No, we have validation to ensure the user cannot get to this point without having at least one hearingDate